libherokubuildpack/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "libherokubuildpack"
+version = "0.10.0"
+edition = "2021"
+rust-version = "1.60"
+license = "BSD-3-Clause"
+description = "Opinionated common code for buildpacks implemented with libcnb.rs"
+keywords = ["buildpacks", "CNB"]
+repository = "https://github.com/heroku/libcnb.rs"
+documentation = "https://docs.rs/libherokubuildpack"
+readme = "./README.md"
+include = ["src/**/*", "../LICENSE", "../README.md"]
+
+[features]
+default = ["digest", "error", "log", "tar", "toml"]
+download = ["dep:ureq", "dep:thiserror"]
+digest = ["dep:sha2"]
+error = ["log", "dep:libcnb"]
+log = ["dep:termcolor"]
+tar = ["dep:tar", "dep:flate2"]
+toml = ["dep:toml"]
+fs = ["dep:pathdiff"]
+
+[dependencies]
+libcnb = { path = "../libcnb", version = "0.10.0", optional = true }
+flate2 = { version = "1.0.24", optional = true }
```

Comment:
```suggestion
flate2 = { version = "1.0.24", optional = true }
libcnb = { path = "../libcnb", version = "0.10.0", optional = true }
```

Reply:
Sorted dependencies as part of [bdebdbf](https://github.com/heroku/libcnb.rs/pull/495/commits/bdebdbfa79bf3f7fe479d498b1d78b119a9bd7fc)